profile/rwahs.xml

```diff
@@ -7711,111 +7711,193 @@
+        <placement code="ca_objects_EarliestYear">
+          <bundle>ca_objects.EarliestYear</bundle>
         </placement>
-        <placement code="video_format">
-          <bundle>ca_objects.video_format</bundle>
+        <placement code="ca_objects_LatestYear">
+          <bundle>ca_objects.LatestYear</bundle>
         </placement>
-        <placement code="text_format">
-          <bundle>ca_objects.text_format</bundle>
+        <placement code="ca_objects_Materials">
+          <bundle>ca_objects.Materials</bundle>
         </placement>
-        <placement code="image_format">
-          <bundle>ca_objects.image_format</bundle>
+        <placement code="ca_objects_Dimensions">
+          <bundle>ca_objects.Dimensions</bundle>
         </placement>
-        <placement code="gen_format">
-          <bundle>ca_objects.gen_format</bundle>
+        <placement code="ca_objects_Importance">
+          <bundle>ca_objects.Importance</bundle>
         </placement>
-        <placement code="idno">
-          <bundle>ca_objects.idno</bundle>
+        <placement code="ca_objects_SignificanceCriteria">
+          <bundle>ca_objects.SignificanceCriteria</bundle>
+          <settings>
+            <setting name="label" locale="en_AU"/>
+            <setting name="format"><![CDATA[]]></setting>
+            <setting name="delimiter"><![CDATA[; ]]></setting>
+            <setting name="maximum_length"><![CDATA[2048]]></setting>
+            <setting name="filter"><![CDATA[]]></setting>
+            <setting name="sense"><![CDATA[singular]]></setting>
+          </settings>
+        </placement>
+        <placement code="ca_objects_ComparativeCriteria">
+          <bundle>ca_objects.ComparativeCriteria</bundle>
+          <settings>
+            <setting name="label" locale="en_AU"/>
+            <setting name="format"><![CDATA[]]></setting>
+            <setting name="delimiter"><![CDATA[; ]]></setting>
+            <setting name="maximum_length"><![CDATA[2048]]></setting>
+            <setting name="filter"><![CDATA[]]></setting>
+            <setting name="sense"><![CDATA[singular]]></setting>
+          </settings>
+        </placement>
+        <placement code="ca_objects_StatementOfSignificance">
+          <bundle>ca_objects.StatementOfSignificance</bundle>
         </placement>
-        <placement code="language">
-          <bundle>ca_objects.language</bundle>
+        <placement code="ca_objects_HistoricalDetails">
+          <bundle>ca_objects.HistoricalDetails</bundle>
         </placement>
-        <placement code="relation">
-          <bundle>ca_objects.related.preferred_labels</bundle>
+        <placement code="ca_objects_ResearchBy">
+          <bundle>ca_objects.ResearchBy</bundle>
         </placement>
-        <placement code="georeference">
-          <bundle>ca_objects.georeference</bundle>
+        <placement code="ca_objects_BibliographReferences">
+          <bundle>ca_objects.BibliographReferences</bundle>
         </placement>
```

Comment:
I didn't see the "references" field appear in the report. Is that because the data is missing from the sample item that the report was run against?